walk/config.go

```diff
@@ -104,6 +116,35 @@ func (cr *Configurer) Configure(c *config.Config, rel string, f *rule.File) {
 	c.Exts[walkName] = wcCopy
 }
 
+func (c *Configurer) loadBazelIgnore(repoRoot string, wc *walkConfig) error {
+	ignorePath := path.Join(repoRoot, ".bazelignore")
+	file, err := os.Open(ignorePath)
+	if os.IsNotExist(err) {
```

Comment:
```suggestion
	if errors.Is(err, fs.ErrNotExist) {
```
New style check recommended by https://pkg.go.dev/os#IsNotExist